ci/src/main/scala/org/typelevel/sbt/TypelevelCiPlugin.scala

```diff
@@ -21,14 +21,15 @@ import org.typelevel.sbt.gha.GenerativePlugin
 import org.typelevel.sbt.gha.GitHubActionsPlugin
 import org.typelevel.sbt.gha.GenerativePlugin.autoImport._
 import com.typesafe.tools.mima.plugin.MimaPlugin
+import scala.language.experimental.macros
 
 object TypelevelCiPlugin extends AutoPlugin {
 
   override def requires = GitHubActionsPlugin && GenerativePlugin && MimaPlugin
   override def trigger = allRequirements
 
   object autoImport {
-    def tlCrossRootProject: CrossRootProject = CrossRootProject()
+    def tlCrossRootProject: CrossRootProject = macro CrossRootProjectMacros.crossRootProjectImpl
```

Comment:
Note that this causes a MiMa issue as this `def` technically no longer exists

Reply:
Should I add an exclusion or do you know of a good workaround for problems like this?